src/components/Authentication/LoginPage.js

```diff
@@ -42,12 +46,27 @@ const GET_USER_TYPE = gql`
     }
 `;
 
+const VERIFY_GOOGLE_OAUTH = gql`
+    query verifyGoogleOauth($loginEmail: String!, $oauthEmail: String!) {
+        verifyGoogleOauth(loginEmail: $loginEmail, oauthEmail: $oauthEmail) {
+            token
+            verified
+        }
+    }
+`;
+
 const LoginPage = () => {
     const history = useHistory();
     const { state } = useLocation();
     const dispatch = useDispatch();
-    const { token, attemptedLogin } = useSelector(({ auth }) => auth);
+    const { token, google_courses } = useSelector(({ auth }) => auth);
     const [userType, setUserType] = useState('');
+    const [googleAuthEnabled, setGoogleAuthEnabled] = useState(false);
+    const [
+        verifyGoogleOauthTokenStatus,
```

Comment:
this sounds like a function - rename to a true/false statement `isGoogleOauthTokenStatusVerified`